renovate.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended"
+  ],
+  "ignorePaths": [],
```

Comment:
```suggestion
```

nit. The default value, `["**/node_modules/**", "**/bower_components/**"]`, seems good enough for us. Do we have to clean the `ignorePaths`?